generator-elide/generators/app/prompt.js

```diff
@@ -0,0 +1,76 @@
+const generator = require('./generator');
+const model = require('./model');
+const misc = require('./misc.js');
+
+// Main prompt when running 'yo elide'
+const prompting = (yo) => {
+  yo.prompt([{
+    type: 'list',
+    name: 'command',
+    message: 'Choose an option below',
+    choices: [ 'Try an example', 'Create a new project', 'Info & Contact' ]
+  }]).then((answers) => {
+    switch (answers.command) {
```

Comment:
It would be nice to have a `default` that gives some sort of error here back to the user. That will allow users to report any issues back up the chain so we can investigate if anything goes wrong.

I believe error'ing out would be the best option since it should *never* happen afaict. If it does, it's something we need to fix.